shared/chat/conversation/attachment-fullscreen/index.desktop.js

```diff
@@ -9,7 +9,10 @@ import {
   ProgressBar,
   ProgressIndicator,
 } from '../../../common-adapters'
-import {FloatingMenuParentHOC, type FloatingMenuParentProps} from '../../../common-adapters/floating-menu'
+import {
+  FloatingMenuParentHOC,
+  type FloatingMenuParentProps,
+} from '../../../common-adapters/floating-menu/parent-hoc'
```

Comment:
With some many files in common-adapters, it seemed like a good idea to start putting components into folders. I also split up the floating menu component and the parent HOC into two separate files for easier readability.

Reply:
Probably, considering we export other HOCs from `common-adapters/index.js`. Not sure why `FloatingMenuParentHOC` never got that same treatment.